--- a/nwg_shell/skel/config/nwg-panel/preset-0
+++ b/nwg_shell/skel/config/nwg-panel/preset-0
@@ -24,7 +24,6 @@
       "clock"
     ],
     "modules-right": [
-      "executor-updates",
       "tray",
       "button-help"
     ],
@@ -127,20 +126,6 @@
       "css-name": "clock",
       "root-css-name": "root-clock"
     },
-    "executor-updates": {
-      "script": "sway-check-updates",
-      "interval": 900,
-      "icon-size": 16,
-      "on-left-click": "foot sway-update",
-      "tooltip-text": "",
-      "on-middle-click": "",
-      "on-right-click": "",
-      "on-scroll-up": "",
-      "on-scroll-down": "",
-      "css-name": "",
-      "root-css-name": "",
-      "icon-placement": "left"
-    },
     "sway-taskbar": {},
     "sway-workspaces": {},
     "scratchpad": {},

--- a/nwg_shell/skel/config/nwg-panel/preset-1
+++ b/nwg_shell/skel/config/nwg-panel/preset-1
@@ -26,7 +26,6 @@
       "executor-cpuav",
       "executor-cpubar",
       "scratchpad",
-      "executor-updates",
       "tray",
       "button-help"
     ],
@@ -123,20 +122,6 @@
       "on-scroll-down": "",
       "css-name": "clock"
     },
-    "executor-updates": {
-      "script": "sway-check-updates",
-      "interval": 900,
-      "icon-size": 16,
-      "on-left-click": "foot sway-update",
-      "tooltip-text": "",
-      "on-middle-click": "",
-      "on-right-click": "",
-      "on-scroll-up": "",
-      "on-scroll-down": "",
-      "css-name": "",
-      "root-css-name": "",
-      "icon-placement": "left"
-    },
     "width": "auto",
     "sway-workspaces": {
       "numbers": [

--- a/nwg_shell/skel/config/nwg-panel/preset-2
+++ b/nwg_shell/skel/config/nwg-panel/preset-2
@@ -23,7 +23,6 @@
     "modules-right": [
       "executor-cpuavg",
       "executor-cpubar",
-      "executor-updates",
       "clock",
       "tray",
       "button-help"
@@ -140,20 +139,6 @@
       "css-name": "clock",
       "root-css-name": "root-clock"
     },
-    "executor-updates": {
-      "script": "sway-check-updates",
-      "interval": 900,
-      "icon-size": 16,
-      "on-left-click": "foot sway-update",
-      "tooltip-text": "",
-      "on-middle-click": "",
-      "on-right-click": "",
-      "on-scroll-up": "",
-      "on-scroll-down": "",
-      "css-name": "",
-      "root-css-name": "",
-      "icon-placement": "left"
-    },
     "sway-workspaces": {},
     "scratchpad": {
       "css-name": "",

--- a/nwg_shell/skel/config/nwg-panel/preset-3
+++ b/nwg_shell/skel/config/nwg-panel/preset-3
@@ -23,7 +23,6 @@
     ],
     "modules-right": [
       "scratchpad",
-      "executor-updates",
       "tray",
       "button-help"
     ],
@@ -114,20 +113,6 @@
       "on-scroll-down": "",
       "css-name": "clock"
     },
-    "executor-updates": {
-      "script": "sway-check-updates",
-      "interval": 900,
-      "icon-size": 16,
-      "on-left-click": "foot sway-update",
-      "tooltip-text": "",
-      "on-middle-click": "",
-      "on-right-click": "",
-      "on-scroll-up": "",
-      "on-scroll-down": "",
-      "css-name": "",
-      "root-css-name": "",
-      "icon-placement": "left"
-    },
     "sway-workspaces": {
       "numbers": [
         "1",

--- a/nwg_shell/skel/config/sway/config
+++ b/nwg_shell/skel/config/sway/config
@@ -36,10 +36,9 @@
 
 ###############     Autostart      ##################
 
-exec systemctl --user import-environment DISPLAY WAYLAND_DISPLAY SWAYSOCK
 exec hash dbus-update-activation-environment 2>/dev/null && \
-    dbus-update-activation-environment --systemd DISPLAY WAYLAND_DISPLAY SWAYSOCK
-exec /usr/lib/polkit-gnome/polkit-gnome-authentication-agent-1
+    dbus-update-activation-environment --session DISPLAY WAYLAND_DISPLAY SWAYSOCK
+exec /usr/libexec/polkit-gnome-authentication-agent-1
 exec ~/.azotebg
 
 ### Turn off screen on lid closed

