--- a/nwg_shell_config/shell/custom
+++ b/nwg_shell_config/shell/custom
@@ -43,8 +43,8 @@
   "gtklock-userinfo-under-clock": false,
   "gtklock-powerbar-show-labels": true,
   "gtklock-powerbar-linked-buttons": false,
-  "gtklock-reboot-command": "systemctl reboot",
-  "gtklock-poweroff-command": "systemctl -i poweroff",
+  "gtklock-reboot-command": "loginctl reboot",
+  "gtklock-poweroff-command": "loginctl poweroff",
   "gtklock-playerctl-art-size": 64,
   "gtklock-playerctl-position": "top-right",
   "gtklock-playerctl-show-hidden": true

