--- a/nwg_shell_config/locker.py
+++ b/nwg_shell_config/locker.py
@@ -61,9 +61,9 @@
     "gtklock-logout-command": "swaymsg exit",
     "gtklock-playerctl": False,
     "gtklock-powerbar": False,
-    "gtklock-poweroff-command": "systemctl -i poweroff",
-    "gtklock-reboot-command": "systemctl reboot",
-    "gtklock-suspend-command": "systemctl suspend",
+    "gtklock-poweroff-command": "loginctl poweroff",
+    "gtklock-reboot-command": "loginctl reboot",
+    "gtklock-suspend-command": "loginctl suspend",
     "gtklock-time-format": "%H:%M:%S",
     "gtklock-userinfo": False,
     "gtklock-userswitch-command": ""

